src/components/ClaimReturnedDialog/ClaimReturnedInfo.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Button,
+  Layout,
+  TextArea,
+  Row,
+  Col,
+} from '@folio/stripes/components';
+import { stripesConnect } from '@folio/stripes/core';
+
+import SafeHTMLMessage from '@folio/react-intl-safe-html';
+
+import css from './ClaimReturnedDialog.css';
+
+class ClaimReturnedInfo extends React.Component {
```

Comment:
It seems, this class is very similar to DeclareLostInfo.js. Could you please clarify if there are opportunities/reasons to create a shared component?

Reply:
Thanks for raising this question. Since the method `submit ...` is fundamentally different by the parameters passed in the POST request, refactoring will require significant efforts not provided for by scope of this issue.

Reply:
It seems, the submit and the connection logic can be moved to the parent component.